fuse/pathfs/pathfs.go

```diff
@@ -210,16 +198,23 @@ func (fs *PathNodeFs) Root() nodefs.Node {
 	return fs.root
 }
 
+type nameAndParent struct {
+	name   string
+	parent *pathInode
```

Comment:
this structure implies an invariant. Do you have a verification method that will check the invariant paranoically?